---
layout: page
title: Research Statement
section: CV
top: Chathura Gunasekara
---

Research Statement for Chathura Gunasekara
==================

Introduction
------------

  The rapid increase of genotype and phenotype data have created a need for efficient technologies in moving, storing, analysing and representing them in meaningfull way. Specially with the advent of whole genome approach creates data and we need store and analyse them with computatioanl framworks. My research interest have been revolving around developing computational infrastructure in terms of algorithmic developmetn, web application development and apply and devlop machine learning algorithms. The web applications were developed under the umbrella of plant biology and each web application used different type of data available to the plant biologist. Before starting my PhD, while in Sri Lanka, I worked on a research project to integrate sensor information from coastal transponders around the country to display and visualized on a centalized map. I have used GoogleMap api and other open source GIS applications such as OpenCPN to display realtime sensor information gathered over a wide area network.
  

Past & Current Research projects
---------------------------

My first job as a softwera/Research Engineer, sensor data fution amd modelling accessing multiple sensors to aggregate in realtime and plot in googlemap api. web, internet related stuff, java, php googlemap API, networking stuff, storing in relational databases. 

Biology/Bioinformatics experience is essential as well as previous experience with web/database development. The applicant should have strong knowledge of Java and JavaScript (jQuery). Experience with Linux/Unix, scripting languages (Python), and relational databases.

Popler gene expression data analysis pipeline, perl, php first bioinformatics web application. published in plant and Tree journal. second exact search, c++ backend algorithm, connect to phytozome api to download data as user reguests. front web development.collaborate with phytozome system admin to learn about there api. talking with biologists. presenting posters.

blossom web tool, rna seq data. jquery, perl blast, work in a team of master student. Jbrowse Sam. Bam, Tophat ect

linux server management, apache,mysql ,php

machine learing backend, web based bioinformatics tools...ect
Biology/Bioinformatics experience is essential as well as previous experience with web/database development. The applicant should have strong knowledge of Java and JavaScript (jQuery). Experience with Linux/Unix, scripting languages (Python), and relational databases.
presentation to biologiest. 




Future Research
------------------------


Biology/Bioinformatics experience is essential as well as previous experience with web/database development. The applicant should have strong knowledge of Java and JavaScript (jQuery). Experience with Linux/Unix, scripting languages (Python), and relational databases.

https://wiki.galaxyproject.org/Events/GCC2015?action=AttachFile&do=get&target=S6_T4_EmilyGrau_Galaxy2015.pdf
http://gmod.org/wiki/Tripal
(http://www.cyverse.org/).

The successful candidate will work as part of a small interdisciplinary team of bioinformaticians and data curators. Tasks will include development of the existing database, developing connections to collaborating databases, improving the user experience (interface development) and implementation of novel analysis and visualization tools for comparative genomics. The Postdoctoral Scholar will be provided with training opportunities and will attend conferences to present/train users on these new tools. In addition, publications on innovative software solutions will result from this work.

The Plant Computational Genomics laboratory at University of Connecticut (Storrs, CT) has an opening for a database and software developer position as part of a collaborative project involving researchers from around the world. This person will take a lead role in the continued development of the TreeGenes genomics database and the CartograTree application (treegenesdb.org and cartogratree.org). These web-based resources serve vast amounts of genomic, phenotypic, and environmental data to our diverse user community. In specific, the CartograTree application provides a portal for association genetics studies of georeferenced forest trees through the implementation of web services and high performance computing resources available through the iPlant/Cyverse project (http://www.cyverse.org/). In addition, we are committed to developing mechanisms for data sharing among partner databases through Galaxy modules, the Tripal platform (http://www.tripal.info/), the GMOD project, and custom analysis/visualization platforms.


I want to be a researcher to take part of the computational revolution taking part in the biological research. 
  Internet related technology space is developing in very rapid phase and life science computational infrastructure should keep up with those Developments. A limitation I realized in computational research is, lots of outstanding algorithms and methods are locked away in research papers, scripts archived in zip files or github repositories. For some researchers to use those findings, they have to install, configure and setup the methodology by them self. This has been a limiting factors because it takes far too much time and knowledge and resourses might not available for everyone. Amazon web services somewhat address these issues, but still the amount of work to be done to get something up and running is very large. The obvious next step has been taken by cyVerse.org is the step in right direction. Implementation of the algorithms can be accesible as a cyVerse app, where uses can make use of the high performance computing resources without the overhead of managing them. 
  With the deluge of biological data, computational infrastruture to store, share, and analyze data is just as important as physical infrastucture such as laboratories, DNA sequencing machines, greenhouses that make it possible for wet lab biologist to collect data. The development of computing in the cloud will empower the life science researchers in the use of high performance computing and large data resources.
  Content management systems (CMS) used to be light waight web pages that host several gigabytes of content, now have to deals with data the size of hundrads terabytes or petabytes. With powerfull web technologies such as Drupal have make it possible to manage this type of content in the web, giving users the accessibility of managing large amount of content and analyse in a high performance clound infrasturure. The Tripal is specially customized to use as a front end for biological data management. This can be further customized for each researchers data type and analsys requirements. So building custom interfaces and visualizations to show the analsys results is needed to be done.ability to search and edit content, detail pages for different types of content, customizable page layouts, and so on--to provide users with an intuitive, flexible interface to a database. 
  
The  
   
  Computatioanl infrastructure development for scientific analysis. Amazon web servises for life science. With batteries included. develop consoles to interact with the computatiaonl infractructure for each uses specific need.Tasks will include development of the existing database, developing connections to collaborating databases, improving the user experience (interface development) and implementation of novel analysis and visualization tools for comparative genomics. The Postdoctoral Scholar will be provided with training opportunities and will attend conferences to present/train users on these new tools. In addition, publications on innovative software solutions will result from this work. Training biologist on computatioanl infracsture. conduct workshops ect. Using cyberinfrastructure, teams of researchers can attempt to answer questions that previously were unapproachable because the computational requirements were too large, too complex, or simply unknown to some researchers.

CyVerse cyberinfrastructure includes:

A data storage facility
An interactive, web-based, analytical platform
Cloud infrastructure to use remote servers for computation, analysis, and storage
Web authentication and security services
Support for scaling computational algorithms to run on large, high-speed computers
Education and training in how to use cyberinfrastructure
People with expertise in all of the above

 bottleneck in biological research, namely the inability of traditional computing technology to meet computational challenges and support the cross-discipline collaboration needed to solve todayâ€™s most pressing questions in plant sciences.
 
Answering the need of an era of data science, CyVerse makes broadly applicable computational resources available across the life sciences.

tackle questions surrounding biological data analysis, data integration, and data sharing.[^1]


References
----------

[^1]: Kyle E Niemeyer, Chih-Jen Sung, and Mandhapati P Raju. Skeletal mechanism generation for surrogate fuels using directed relation graph with error propagation and sensitivity analysis. *Combust. Flame* 157(9):1760--1770, 2010. doi:[10.1016/j.combustflame.2009.12.022](http://dx.doi.org/10.1016/j.combustflame.2009.12.022).
