---
layout: page
title: Research Statement
section: CV
top: Chathura Gunasekara
---

Research Statement for Chathura Gunasekara
==================

Introduction
------------

My research interest have been revolving around computational algorithm development, web application development, apply and devlop machine learning
algorithms for constructuing gene regulatory networks using gene expression data. Biology has been in the forfront of computational revolution
given the advent of methods to genereate large amount of data. This have give unpresidented access to the insigts of biological systems unlike in
any time of history and allows us to do biologiciall experiments in a completely diffrerent way. I believe this is the most exciting time to be in
computational biology.


Past Research and Experience
---------------------------

Current Research
--------------



Intended Future Research
------------------------

I want to be a researcher to take part of the computational revolution taking part in the biological research. 
  Internet related technology space is developing in very rapid phase and life science computational infrastructure should keep up with those Developments. A limitation I realized in computational research is, lots of outstanding algorithms and methods are locked away in research papers, scripts archived in zip files or github repositories. For some researchers to use those findings, they have to install, configure and setup the methodology by them self. This has been a limiting factors because it takes far too much time and knowledge and resourses might not available for everyone. Amazon web services somewhat address these issues, but still the amount of work to be done to get something up and running is very large. The obvious next step has been taken by cyVerse.org is the step in right direction. Implementation of the algorithms can be accesible as a cyVerse app, where uses can make use of the high performance computing resources without the overhead of managing them. 
  With the deluge of biological data, computational infrastruture to store, share, and analyze data is just as important as physical infrastucture such as laboratories, DNA sequencing machines, greenhouses that make it possible for wet lab biologist to collect data. The development of computing in the cloud will empower the life science researchers in the use of high performance computing and large data resources.
  Content management systems (CMS) used to be light waight web pages that host several gigabytes of content, now have to deals with data the size of hundrads terabytes or petabytes. With powerfull web technologies such as Drupal have make it possible to manage this type of content in the web, giving users the accessibility of managing large amount of content and analyse in a high performance clound infrasturure. The Tripal is specially customized to use as a front end for biological data management. This can be further customized for each researchers data type and analsys requirements. So building custom interfaces and visualizations to show the analsys results is needed to be done.ability to search and edit content, detail pages for different types of content, customizable page layouts, and so on--to provide users with an intuitive, flexible interface to a database. 
  
The  
   
  Computatioanl infrastructure development for scientific analysis. Amazon web servises for life science. With batteries included. develop consoles to interact with the computatiaonl infractructure for each uses specific need.Tasks will include development of the existing database, developing connections to collaborating databases, improving the user experience (interface development) and implementation of novel analysis and visualization tools for comparative genomics. The Postdoctoral Scholar will be provided with training opportunities and will attend conferences to present/train users on these new tools. In addition, publications on innovative software solutions will result from this work. Training biologist on computatioanl infracsture. conduct workshops ect. Using cyberinfrastructure, teams of researchers can attempt to answer questions that previously were unapproachable because the computational requirements were too large, too complex, or simply unknown to some researchers.

CyVerse cyberinfrastructure includes:

A data storage facility
An interactive, web-based, analytical platform
Cloud infrastructure to use remote servers for computation, analysis, and storage
Web authentication and security services
Support for scaling computational algorithms to run on large, high-speed computers
Education and training in how to use cyberinfrastructure
People with expertise in all of the above

 bottleneck in biological research, namely the inability of traditional computing technology to meet computational challenges and support the cross-discipline collaboration needed to solve todayâ€™s most pressing questions in plant sciences.
 
Answering the need of an era of data science, CyVerse makes broadly applicable computational resources available across the life sciences.

tackle questions surrounding biological data analysis, data integration, and data sharing.[^1]


References
----------

[^1]: Kyle E Niemeyer, Chih-Jen Sung, and Mandhapati P Raju. Skeletal mechanism generation for surrogate fuels using directed relation graph with error propagation and sensitivity analysis. *Combust. Flame* 157(9):1760--1770, 2010. doi:[10.1016/j.combustflame.2009.12.022](http://dx.doi.org/10.1016/j.combustflame.2009.12.022).
