Chathura Gunasekara
============

-------------------     ----------------------------
2101 Woodmar Dr                     cjgunase@mtu.edu

Apt C Houghton, MI, USA               (213) 316-6296 
-------------------     ----------------------------

Education
---------

2017 Spring (expected)
:   **PhD, Computational Science & Engineering**; Michigan Technological University (Houghton, MI)

      Advisor: [Dr. Hairong Wei](http://www.mtu.edu/forest/about/faculty/wei/)
   
      PhD Thesis title: Bioinformatics Tools and Algorithms Development for Gene Regulatory Network Inference*
       

2010
:   **BS, Computational Physics**; University of Colombo (Sri Lanka)
    
    Major in Computer Science with minor in Physics and Applied Mathematics*

Professional Experience & Publications
----------

**Graduate Research Assistant : 2013 - Present**

_Curruent Projects_

* TF-miner is a data analysis pipeline based on partial least squares and graphical gaussian models to identify regulatory TFs using microarray expression under stress conditions. [[source]](https://github.com/cjgunase/TF-miner)


* Infer gene regulatory network from DREAM5 dataset and  extend the method for arabidopsis gene expr data  at a Bioinformatics Workshop at Noble Foundation, Ardmore, Oklahoma.[[source]](https://github.com/cjgunase/DREAM5_gene_net.git)


* Pairwise analysis of gene expression data to find regulatory TF clusters using novel association methods.[[source]](https://github.com/cjgunase/MIC_based_gene_expr_analysis)


* Currently implementing a  web based gene expression data analysis pipeline to identify Transcription Factor(TF) clusters which associates with known biological pathways.[[source]](https://github.com/cjgunase/TF-Cluster), [[web]](http://sys.bio.mtu.edu/cluster/index.php)

_Completed projects_

* Developed an algorithm and web application to search for degenerate motifs in the promoter regions of 50 plant species genomes.[[source]](https://github.com/cjgunase/exactSearch), [[web]](http://sys.bio.mtu.edu/motif/), [[publication]](http://plantmethods.biomedcentral.com/articles/10.1186/s13007-016-0126-6)


* Configured, installed a JBrowse genome browser to visualize RNA-seq and Ribo-seq of wild-type and STTM mutants to parse fastq files.[[web]](https://blossom.ffr.mtu.edu/designindex2.php)


* Collaborated with a lab member to publish an algorithm to infer hierachical gene regulatory network from gene expression data.[[publication]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4797117/)


* Collaborated with a lab member  to develop the  Poplar Gene Expression Pipeline web application. [[web]](http://sys.bio.mtu.edu), [[publication]](http://link.springer.com/article/10.1007/s11295-014-0745-x#page-1)


* Contributed to the publication of a lab member by creating a Circos Visualization of genomics data from Birch genome.[[source]](https://github.com/cjgunase/myVisualizations/tree/master/circos)

\pagebreak

**Software Engineer/Research Engineer : 2010 - 2013**

I Worked on a [research project](http://www.vidusayura.org/?page_id=2) with University of Colombo School of Computing and Sri Lanka Navy.


* Develop algorithms and to implement using Java and web based technologies a Survailance platfrom to fuse data from multiple transponders such as AIS, RADAR sensors around Sri Lanken coast line.[[Publication]](http://www.icter.org/conference/icter2012/paper/50)

* Maritime Navigation Simulator Project, Low Cost 3D Immersive Telepresence for Survaillance, Planning, Maneuring : 3D-COP	10.5176/2251-1679_CGAT31. Computer Games, Multimedia & Allied Technology Conference 2012.[[Publication]](http://www.cgames.com.sg/PriorYearsPaper2012.html)

* Maritime Navigation Simulator Project for Simulating Narrow Channel Effect on Surge Motion of a Ship in a Virtual Environment. [[Publication]](http://www.icter.org/conference/icter2012/paper/31)

* Undergraduate Research on Spatialized Real Time Auditory Interface for a Virtual Maritime Application in 2010.[[Publication]](http://www.icter.org/conference/archive2011/index.php/icter/ICTer2010/paper/view/65)


Technical Experience and Recent Course work
--------------------

Technical Skills
:   Software engineering (Object oriented programming, version control, Documentation)
:   Python (scikit-learn/numpy/scipy/pandas), R, Microsoft Excel
:   Perl, Java, C++, Database(SQL), Linux/Unix/Shell Scripting
:   Web Development (Linux/Apache/MySql/PHP) and web server management

Data Science
:   Applied Predictive Modelling (Fall 2014), Introduction to Data Science (Fall 2014)
:   Data Mining (Spring 2014), Data mining for geo spatial applications (Fall 2015)
:   Machine Learning Foundations: A Case Study Approach by University of Washington on Coursera [[verify]](Machine Learning Foundations: A Case Study Approach by University of Washington on Coursera)
:   Machine Learning: Regression by University of Washington on Coursera [[verify]](https://www.coursera.org/account/accomplishments/certificate/2AXNSM86GTZS)

Statistics
:   Statistical Methods (Fall 2013), Regression Analysis (Spring 2013)
:   Time series analysis and forcasting (Spring 2015)

Computer Science
:   Advanced Scripting and Programming (Fall 2015)
:   Algorithmic Toolbox by University of California, San Diego & Higher School of Economics on Coursera.[[verify]](https://www.coursera.org/account/accomplishments/certificate/KSF3Q7M96AWB)

Biology
:   Bioinformatics Programming Skills (Fall 2013)
:   Following Genomic Data Science Specialization Johns Hopkins University on Coursera

Professional Development
------------------------

2016
:     Presentation at NSFproject/Bioinformatics workshop at Noble Foundation, Ardmore, OK on Gene regulatory network prediction using partial least squares and bayesian network methods.[[_presentation_]](http://www.slideshare.net/secret/15sqXSMbpOHAOQ)

2016
:     Poster presentation at research symposium of Life Science and Technology Institure, Michigan Tech

2014
:     Presentation at Graduate research symposium organized by Graduate Student Governement, Michigan Tech
