<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */
  
  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }
  
  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  h1:hover {
      background-color: #757575;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  h2:hover {
      background-color: #397249;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Definitions */
  dt {
      float: left;
      clear: left;
      width: 17%;
      /*font-weight: bold;*/
  }
  dd {
      margin-left: 17%;
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }
  
  /* Blockquotes */
  blockquote {
      text-align: center
  }
  
  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Horizontal separators */
  hr {
      color: #A6A6A6;
  }
  
  table {
      width: 100%;
  }
  </style>
</head>
<body>
<h1 id="chathura-gunasekara">Chathura Gunasekara</h1>
<table>
<col width="33%" />
<col width="38%" />
<tbody>
<tr class="odd">
<td align="left">2101 Woodmar Dr</td>
<td align="right">cjgunase@mtu.edu</td>
</tr>
<tr class="even">
<td align="left">Apt C Houghton, MI, USA</td>
<td align="right">906-231-3808</td>
</tr>
</tbody>
</table>
<h2 id="education">Education</h2>
<dl>
<dt>2013 May - 2017 Spring (expected)</dt>
<dd><p><strong>PhD, Computational Science &amp; Engineering</strong>; Michigan Technological University (Houghton, MI)</p>
<p><em>PhD Thesis title: Bioinformatics Tools and Algorithms Development for Gene Regulatory Network Inference</em></p>
</dd>
<dt>2006-2010</dt>
<dd><p><strong>BS, Computational Physics</strong>; University of Colombo (Sri Lanka)</p>
<p><em>Major in Computer Science with minor in Physics and Applied Mathematics</em></p>
</dd>
</dl>
<h2 id="projects-work-experience-publications">Projects, Work Experience &amp; Publications</h2>
<p><strong>Graduate Research Assistant : 2013 - Present</strong></p>
<p><em>Currently I am Working under <a href="http://www.mtu.edu/forest/about/faculty/wei/">Dr. Hairong Wei</a>, conducting research to infer gene regulatory networks and indentify regulatory transcription factors (TFs) which control known biological pathways in <em>Arabidopsis thaliana</em> under stress conditions using gene expression data.</em></p>
<ul>
<li><p><a href="https://github.com/cjgunase/TF-miner">TF-miner</a></p></li>
<li><p>My early <a href="github%20repo">attempts</a> for this research was <a href="http://www.slideshare.net/secret/15sqXSMbpOHAOQ">presented</a> at NSF Project/Bioinformatics Workshop @ Noble Foundation, Ardmore, Oklahoma.</p></li>
<li><p>Pairwise analysis of Pathway - Transcription Factor gene expression data to find regulatory TF clusters.<a href="https://github.com/cjgunase/MIC_based_gene_expr_analysis">Git</a></p></li>
<li><p>Currently implementing a <a href="http://sys.bio.mtu.edu/cluster/index.php">web</a> based gene expression data analysis pipeline to identify Transcription Factor(TF) clusters which associates with known biological pathways using novel pair-wise gene association methods such as Maximum Information Coefficient(MIC), Distance Correlation.</p></li>
</ul>
<p><em>Completed projects</em></p>
<ul>
<li><p><a href="https://github.com/cjgunase/exactSearch">Developed</a>, <a href="http://sys.bio.mtu.edu/motif/">implemented</a> and <a href="http://plantmethods.biomedcentral.com/articles/10.1186/s13007-016-0126-6">published</a> an algorithm and web application to search for degenerate motifs in the promoter regions of 50 plant species genomes. Our suffix tree based search algorithm can search for 100 degerate motifs in 35000 genes under 4 minutes!.</p></li>
<li><p>I configured, installed and developed Perl scripts for parsing the FASTQ files using open sourse tools for a genome browser to <a href="https://blossom.ffr.mtu.edu/designindex2.php">visualize</a> RNA-seq and Ribo-seq of wild-type and STTM mutants. This browser provides information on global translation status and miRNA-directed translation inhibition, which is currently lacking in plants. This web portal make them available to the research community.</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4797117/">Co-authored</a> an algorithm to infer hierachical gene regulatory network from gene expression data.</p></li>
<li><p><a href="http://sys.bio.mtu.edu">Implemented</a> and <a href="http://link.springer.com/article/10.1007/s11295-014-0745-x#page-1">Co-authored</a> Poplar Gene Expression Pipeline web application.</p></li>
<li><p><a href="https://github.com/cjgunase/myVisualizations/tree/master/circos">Contributor</a>, to Birch Genome publication building a Circos Visualization</p></li>
</ul>
<p><strong>Software Engineer/Research Engineer : 2010 - 2013</strong></p>
<p>Worked on a <a href="http://www.vidusayura.org/?page_id=2">collaborative research project</a> with University of Colombo School of Computing and Sri Lanka Navy. Following are the list of publications I authored/contributed:</p>
<ul>
<li><p>Develop algorithms and to implement using Java and web based technologies a Survailance platfrom to fuse data from multiple transponders such as AIS, RADAR sensors around Sri Lanken coast line.<a href="http://www.icter.org/conference/icter2012/paper/50">First-Author</a></p></li>
<li><p>Maritime Navigation Simulator Project, <a href="http://www.cgames.com.sg/PriorYearsPaper2012.html">First-Author CGAT 2012, Thailand</a>, Low Cost 3D Immersive Telepresence for Survaillance, Planning, Maneuring : 3D-COP 10.5176/2251-1679_CGAT31. Computer Games, Multimedia &amp; Allied Technology Conference 2012.</p></li>
<li><p>Maritime Navigation Simulator Project, <a href="http://www.icter.org/conference/icter2012/paper/31">Co-Author ICter 2012</a> for Simulating Narrow Channel Effect on Surge Motion of a Ship in a Virtual Environment</p></li>
<li><p>Undergraduate Research <a href="http://www.icter.org/conference/archive2011/index.php/icter/ICTer2010/paper/view/65">Conference paper</a>, Spatialized Real Time Auditory Interface for a Virtual Maritime Application in 2010.</p></li>
</ul>
<h2 id="technical-experience-and-recent-course-work">Technical Experience and Recent Course work</h2>
<dl>
<dt>Technical Skills</dt>
<dd><p>Software and Programming Languages</p>
<ul>
<li>Perl,Python (scikit-learn, numpy, scipy, pandas), R</li>
<li>Java, C++, Database(SQL), Linux/Unix/Shell Scripting, Microsoft Excel, LaTeX</li>
<li>Web Development in Linux/Apache/MySql/PHP</li>
</ul>
</dd>
<dt>Data Science</dt>
<dd><strong>Applied Predictive Modelling</strong> Fall 2014
</dd>
<dd><strong>Introduction to Data Science</strong> Fall 2014
</dd>
<dd><strong>Data Mining</strong> Spring 2014
</dd>
<dd><strong>Data mining for geo spatial applications</strong> Fall 2015
</dd>
<dd><p><strong>Machine Learning - Regression</strong> Coursera Verified Certification (online)</p>
</dd>
<dt>Statistics</dt>
<dd><strong>Statistical Methods</strong> Fall 2013
</dd>
<dd><strong>Regression Analysis</strong> Spring 2013
</dd>
<dd><p><strong>Time series analysis and forcasting</strong> Spring 2015</p>
</dd>
<dt>Computer Science</dt>
<dd><strong>Advanced Scripting and Programming</strong> Fall 2015
</dd>
<dd><p><strong>Algorithmic Toolbox</strong> Coursera Verified Certification (online)</p>
</dd>
<dt>Bioinformatics</dt>
<dd><p><strong>Bioinformatics Programming Skills</strong> Fall 2013</p>
</dd>
</dl>
</body>
</html>
